src/components/SaveMultiButton.tsx

```diff
@@ -6,126 +6,136 @@
  *
  */
 
+import { useState } from "react";
 import { useTranslation } from "react-i18next";
-import { SerializedStyles, css } from "@emotion/react";
-import styled from "@emotion/styled";
-import { colors } from "@ndla/core";
-import { CheckboxCircleFill } from "@ndla/icons/editor";
-import MultiButton from "./MultiButton";
+import { Portal } from "@ark-ui/react";
+import { ArrowDownShortLine } from "@ndla/icons/common";
+import { CheckLine } from "@ndla/icons/editor";
+import { Button, IconButton, MenuContent, MenuItem, MenuPositioner, MenuRoot, MenuTrigger } from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
+import { SAVE_BUTTON_ID } from "../constants";
 
-type SaveModifiers = "save" | "saving" | "saved";
+const StyledMultiButton = styled("div", {
+  base: { display: "flex" },
+});
 
-const StyledSpan = styled("span")`
-  display: flex;
-  justify-content: space-evenly;
-`;
+const StyledButton = styled(Button, {
+  variants: {
+    composite: {
+      true: {
+        borderRightRadius: "0",
+      },
+    },
+  },
+});
 
-export const saveButtonAppearances: Record<string, SerializedStyles> = {
-  saved: css`
-    &,
-    &:hover,
-    &:disabled {
-      color: white;
-      transition: all 0.5s ease;
-      background-color: ${colors.support.green};
-      border-color: ${colors.support.green};
-    }
-  `,
-  saving: css`
-    &,
-    &:hover,
-    &:disabled {
-      color: white;
-      transition: all 0.5s ease;
-      background-color: ${colors.support.greenLight};
-      border-color: ${colors.support.greenLight};
-    }
-  `,
-};
+const StyledIconButton = styled(IconButton, {
+  base: {
+    borderLeftRadius: "0",
+    borderInlineStart: "1px solid",
+    borderColor: "surface.default",
```

Comment:
Bruke annen farge her?

Reply:
Synes jo egentlig at vi burde det. Kunne eventuelt hatt en gap på 2px mellom de to knappene. Lagre-knappen kan være primary, og menu-knappen kan være secondary?

Reply:
Hørte med Hedvig, vi går for gap på 2px og primary på begge to!